infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/OpenShiftPvcStrategy.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2012-2017 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.workspace.infrastructure.openshift;
+
+/**
+ * Defined set of supported strategies for workspace PVCs.
+ *
+ * @author Anton Korneta
+ */
+public enum OpenShiftPvcStrategy {
```

Comment:
Enum doesn't allow to bind new strategy in custom assembly. Can you rework the code to allow new strategies creation?

Reply:
makes sense I'll rework this